workspaces/ui-v2/src/pages/diffs/EndpointDocumentationPane.tsx

```diff
@@ -108,15 +110,15 @@ export const EndpointDocumentationPane: FC<
           />
         </div>
       </CodeBlock>
-      <div style={{ height: 50 }} />
 
       {thisEndpoint.query && (
         <HighlightedLocation
+          className={classes.bodyContainer}
           targetLocation={highlightedLocation}
           expectedLocation={Location.Query}
         >
           {/* TODO QPB - change id from this to query id from spectacle */}
-          <div className={classes.bodyContainer} id="query-parameters">
+          <div id="query-parameters">
```

Comment:
I assume this `id` is here to allow deep / fragment links to it?

Reply:
That is correct - there's potentially a better way to link these but I haven't thought about this that much

Reply:
Perfectly fine way to link to something deeper on a page, I often hunt people's DOMs for them so my links can be more accurate :)